.github/workflows/publish.yml

```
@@ -65,7 +65,7 @@ jobs:
       BUILDX_NO_DEFAULT_ATTESTATIONS: 1
 
     steps:
-      - uses: actions/checkout@71cf2267d89c5cb81562390fa70a37fa40b1305e # v5.0.0
+      - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v5.0.0
```

Isn't Dependabot meant to update the comment as well? 🤔

```suggestion
      - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0

- name: Create GitHub Release with changelog
env:
@@ -39,7 +39,7 @@ jobs:
id-token: write # Required for OIDC in NPM Trusted Publishing

steps:
      - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0

- uses: actions/setup-node@2028fbc5c25fe9cf00d9f06a71cc4710d4507903 # v6.0.0
with:
@@ -65,7 +65,7 @@ jobs:
BUILDX_NO_DEFAULT_ATTESTATIONS: 1

steps:
      - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
```